### ISSUE TITLE
Arduino Nano Every  in ATMEGA328 compability mode - A7 works as A6 and vice versa.

### ISSUE THREAD

**Issue by elektros230** [arduino/ArduinoCore-megaavr#38]
In Arduino nano every seems to be changed pinouts of PIN_A7 and PIN_A6 in ATMEGA328 compability mode.
A7 works as A6 and vice versa.

**Comment by elektros230** [comment]
Now I solve that bug by adding compiler directive:

```
#ifdef FIX_BUG_NANO_EVERY
#define outVoltagePin PIN_A7 //A7 as real A6 pin output
#else
#define outVoltagePin PIN_A6
#endif
```

**Comment by facchinm** [comment]
Hi @elektros230 , thanks for reporting! I fixed the issue in  3737f45 , it will be included in upcoming core 1.8.3 release
